align_method: "VGICPCUDA"
voxelgrid_leaf: 0.1

pcl_icp:
  max_iterations: 50
  max_correspondence_distance: 1.0
  transformation_epsilon: 1e-6
  euclidean_fitness_epsilon: 1e-6
  use_reciprocal_correspondences: true

pcl_ndt:
  step_size: 0.1
  voxel_resolution: 1.0
  transformation_epsilon: 1e-6
  max_iterations: 40

gicp:
  k_correspondences: 20
  max_correspondence_distance: 2.0
  num_threads: 8

vgicp:
  k_correspondences: 20
  voxel_resolution: 1.0
  neighbor_search_method: "DIRECT1" # DIRECT1 DIRECT7 DIRECT27 DIRECT_RADIUS
  num_threads: 8

vgicp_cuda:
  k_correspondences: 20
  voxel_resolution: 1.0
  neighbor_search_method: "DIRECT1"
  neighbor_search_radius: 1.5
  nearest_neighbor_method: "CPU_PARALLEL_KDTREE" # CPU_PARALLEL_KDTREE, GPU_BRUTEFORCE, GPU_RBF_KERNEL
  kernel_width: 0.5

ndt_cuda:
  voxel_resolution: 1.0
  neighbor_search_method: "DIRECT1"
  neighbor_search_radius: 1.5
  ndt_distance_mode: "P2D"
